contracts/libraries/Message.sol

```diff
@@ -90,14 +90,16 @@ library Message {
         return keccak256(abi.encodePacked(prefix, msgLength, message));
     }
 
+    // It is not necessary to check that arrays have the same length since it will be handled
```

Comment:
Consider to move the comment just before the line containing `require(_vs.length >= requiredSignatures);`

Reply:
Fixed https://github.com/poanetwork/poa-bridge-contracts/pull/204/commits/f1fc07bb5ca88eb137023c3b1446739b1f108bf5